bulk_pcap.py

```diff
@@ -0,0 +1,56 @@
+import os
```

Comment:
```suggestion
#!/usr/bin/env python3

import os
```

Reply:
I will batch this and the rest of the suggestions once @hwh33 looks over the changes.